### PR TITLE
Add fuzzing handbook

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Fixes: https://github.com/cncf/tag-security/issues/1080

The handbook is in a state with a lot of information and guides on fuzzing, which can be used during fuzzing engagements by the CNCF and for the maintainers of projects. I think there are some areas that can be improved and extended, and this will also be an ongoing effort, and I assume the handbook will also evolve as the fuzzing efforts across CNCF progress.

I put a somewhat placeholder as title page, and am working on something a bit better. I'd like to keep the ability to generate the PDF through scripts as this makes it a lot easier to update.

In addition to the artifact itself, we will also be using this for the a fuzzing session at the upcoming KubeCon https://kccncna2023.sched.com/event/1TxW4

### CODE REVIEW

**Review by anvega — APPROVED** [review]

**Comment by caniszczyk** [comment]
LGTM this is a great addition!

File: security-fuzzing-handbook/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+docker run --rm \
+             --volume "$(pwd):/data" \
+	     --user $(id -u):$(id -g) \
+	     pandoc/extra fuzzing-handbook.md \
+	     -o handbook-fuzzing.pdf \
```
Comment (makkes) [inline security-fuzzing-handbook/build.sh:6]: Either
```suggestion
	     pandoc/extra handbook-fuzzing.md \
	     -o handbook-fuzzing.pdf \
```
or
```suggestion
	     pandoc/extra fuzzing-handbook.md \
	     -o fuzzing-handbook.pdf \
```

Comment (DavidKorczynski) [reply security-fuzzing-handbook/build.sh:6]: It's intentional as is, and I'd prefer to keep it as is -- it makes it easier to use the tab on the command line.

**Review by anvega — APPROVED** [review]

**Comment by anvega** [comment]
@DavidKorczynski Thank you all for your hard work and for sharing your knowledge so generously.  It's clear that a tremendous amount of effort and expertise has gone into this contribution. The technical depth, the attention to detail, and the clarity of the writing will undoubtedly make a significant impact on those who seek to learn and master fuzzing techniques.

Congratulations to everyone involved for a job exceptionally well done!

**Comment by JonZeolla** [comment]
Awesome work, thank you for this David and Adam!

**Pull request merged**